geonetwork/variant-postgres.md

```diff
@@ -8,28 +8,31 @@ In order to setup the connection from geonetwork, you **must** inject the follow
 
 If your postgres instance is listening on a non-standard port, you must also set that variable: - `POSTGRES_DB_PORT`: postgres port on your database server (defaults to `5432`)
 
-### Linking to a postgres container
+### Connecting to a postgres database
 
-Linking to a postgres container, is pretty straightforward: - `--link <some-postgres-container>:postgres`
+If you want to connect to a postgres server, you need to pass an extra environment variable, `POSTGRES_DB_HOST`,
+containing the address of this server.
```

Comment:
The rest of the content seems fine to me, just need to combine these two lines (13-14) to make markdownfmt and the TravisCI test happy.

Reply:
[Done](https://github.com/GeoCat/docs/commit/9013c0dddc6ba169c47b1da1104f644abe15e7f1)!